src/main/java/nextstep/courses/domain/Base.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.domain;
+
+import java.time.LocalDateTime;
+
+public class Base {
```

Comment:
```suggestion
public abstract class Base {
```
추상 클래스로 구현해 직접 생성할 수 없도록 구현

Reply:
추상 클래스로 변경하여 컴파일러가 직접 생성할 수 없게끔 보장하게 변경하고 오직 직접 생성용으로 한다
결국 공통상태 관리하는 역할로 둔다.